Makefile

```diff
@@ -58,19 +57,17 @@ busybox.tgz:
 	docker save busybox:latest | gzip -c > busybox.tgz
 
 calico-node.tgz:
-	docker pull calico/node:latest
-	docker save calico/node:latest | gzip -c > calico-node.tgz
+	docker pull calico/node:v0.10.0
```

Comment:
This should not be pinned, surely?  Doesn't it mean the STs won't test the latest code?